src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertRobot = {};
+  let prewPseudoVallue = '';
```

Comment:
No need use `pseudo` and you should fix typo in word `value` and `prev` (shortand from previous word)
```suggestion
  let prevValue = '';
```